core/WFCORE-2806_Enable_resolution_of_standard_server_system_properties_on_domain_mode.adoc

```diff
@@ -0,0 +1,129 @@
+    "result" => undefined,
+    "server-groups" => {"main-server-group" => {"host" => {"master" => {"server-one" => {"response" => {
+        "outcome" => "success",
+        "response-headers" => {
+            "operation-requires-restart" => true,
+            "process-state" => "restart-required"
+        }
+    }}}}}}
+}
+
+[domain@localhost:9990 /] /server-group=main-server-group:restart-servers
+{
+    "outcome" => "failed",
+    "failure-description" => {"domain-failure-description" => "WFLYCTL0158: Operation handler failed: java.lang.IllegalStateException: WFLYCTL0211: Cannot resolve expression '-Xloggc:${jboss.server.log.dir}/gc.log'"},
+    "rolled-back" => true
+}
+----
+
+The reason is the standard management model resolution resolves these expressions by looking at the JVM environment properties. These properties are created when the HC configures the managed servers and are available for resolution only on the server JVM.
+
+This situation forces the users to configure per server the JVM options figuring out the name of the server name:
+
+----
+[domain@localhost:9990 /] /host=master/server-config=server-one/jvm=default:add-jvm-option(jvm-option=-Xloggc:${jboss.domain.servers.dir}/server-name/log/gc.log)
+----
+
+This approach is inappropriate if you have a considerable amount of servers, it doesn't scale well and doesn't take into account if the HC server grouping is by-type or by-server.
+
+The goal of this RFE is to make available some of these server configuration properties to configure the managed server JVMs.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-2806[WFCORE-2806]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-823[EAP7-823]
+* https://issues.redhat.com/browse/WFLY-15095[WFLY-15095]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:jjedlick@redhat.com[Jakub Jedlicka]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+wildfly-core
+
+=== Other Interested Projects
+
+N/A
+
+=== Relevant Installation Types
+
+* [ ] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [ ] OpenShift s2i
+
+* [ ] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
+
+* The following standard server system properties will be available as management model expressions to configure the managed server JVM:
+** jboss.server.base.dir: Root directory for an individual server instance.
+** jboss.server.data.dir: Directory the server will use for persistent data file storage.
+** jboss.server.log.dir: Directory the server will use for log file storage.
+** jboss.server.temp.dir: Directory the server will use for temporary file storage.
+* These environment variables will be available to configure the jvm resources at:
```

Comment:
I'd say

"The use of these properties to resolve expressions will be available for attributes in the jvm resources at:"

Reply:
Done. Thanks for reviewing it.